components/host-dashboard/reports-section/TransactionsOverviewSection.js

```diff
@@ -233,6 +240,14 @@ const TransactionsOverviewSection = ({ host, isLoading }) => {
     series[1].data.sort((a, b) => new Date(a.x) - new Date(b.x));
   }
```

Comment:
I think this part needs to be modified along with the changes I suggested above so that we handle for the missing data points within the third array of the `series` (`series[2]`). Otherwise we will show `NaN` for all missing data points. 

![test](https://user-images.githubusercontent.com/12435965/164605130-58548da3-1d11-4136-bb57-b319ae336669.png)

Reply:
Good catch :+1: I've pushed an improvement in https://github.com/opencollective/opencollective-frontend/pull/7715/commits/90c540787a0e9fa72bf0f613dbf9022effd8b442 to make this code work regardless of the number of series.